.dockerignore

```diff
@@ -1,4 +1,24 @@
-.git
+.git/
 Makefile.eve
 Dockerfile.gcc
```

Comment:
I really like changes here, but they are not applied only for this PR, please, put them into a dedicated commit and provide them for other branches as well, including arm64 ones....

Reply:
@rene Done! I also moved a fix for `CONFIG_BPF_KPROBE_OVERRIDE` check into separate commit. Now both can be backported to ARM branches